plio/management/commands/pliooauth2application.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management.base import BaseCommand
+import os
+from oauth2_provider.models import Application
+
+
+class Command(BaseCommand):
+    help = "Creates OAuth2 client credentials"
+
+    def handle(self, *args, **options):
+        default_oauth2_client_setup = os.environ.get(
+            "DEFAULT_OAUTH2_CLIENT_SETUP", False
+        )
+        default_oauth2_client_id = os.environ.get("DEFAULT_OAUTH2_CLIENT_ID")
+        default_oauth2_client_secret = os.environ.get("DEFAULT_OAUTH2_CLIENT_SECRET")
+        if default_oauth2_client_setup:
```

Comment:
we can also rewrite it as:
```python
if not default_oauth2_client_setup:
    print("Default OAuth2 client setup is disabled. Skipping.")
    return
.
.
.
```
and so on for the other `if-else` blocks, right?

This will avoid a lot of branching in the code and early exit - also, helps with readability IMO.

Reply:
makes sense. yes. will do.